lib/payload.json

```
@@ -212,9 +212,19 @@
         "action": {
           "type": "object",
           "properties": {
-            "motionState": {
-              "type": "number",
-              "description": "State of Motion (number)"
+            "motion": {
+              "type": "object",
+              "properties": {
+                "motionState": {
```

```suggestion
                "detected": {
```

We're already in `motion` so no need to have `motion.motionXXX`